packages/js/data/src/settings/actions.ts

```diff
@@ -37,56 +48,59 @@ export function updateErrorForGroup( group, data, error, time = new Date() ) {
 	};
 }
 
-export function setIsRequesting( group, isRequesting ) {
+export function setIsRequesting( group: string, isRequesting: boolean ) {
 	return {
 		type: TYPES.SET_IS_REQUESTING,
 		group,
 		isRequesting,
 	};
 }
 
-export function clearIsDirty( group ) {
+export function clearIsDirty( group: string ) {
 	return {
 		type: TYPES.CLEAR_IS_DIRTY,
 		group,
 	};
 }
 
-// allows updating and persisting immediately in one action.
-export function* updateAndPersistSettingsForGroup( group, data ) {
```

Comment:
Move this function after `persistSettingsForGroup` to fix `@typescript-eslint/no-use-before-define`.